src/main/java/io/github/jpmorganchase/fusion/api/operations/FusionAPIUploadOperations.java

```diff
@@ -371,6 +407,12 @@ private FusionAPIUploadOperationsBuilder uploadThreadPoolSize(int uploadThreadPo
             this.uploadThreadPoolSize = uploadThreadPoolSize;
             return this;
         }
+
+        @SuppressWarnings("PIT")
```

Comment:
What's this suppression for?

Reply:
Discussed with Ian, false positives from PIT. Needs to be addressed in a future PR

Reply:
Will be addressing this in next batch of build work.  False metric being provided here.